README.md

```diff
@@ -4,29 +4,68 @@ This repo hosts the distributable production version of the A2J Viewer.
 
-While other server environments may work, they have not been tested.  Should you get another hosting environment working, please do a Pull Request at the hosted [A2J Viewer](https://github.com/CCALI/A2JViewer) repo to let us know any steps taken so that we may share with others.
+While other server environments may work, they have not been tested.  Should you get another hosting environment working, please do a Pull Request at the hosted [A2J Viewer](https://github.com/CCALI/a2jviewer) repo to let us know any steps taken so that we may share with others.
 
-## Installing
-Unzip the viewer package into your webroot or preferred directory on your web server. The Apache user, for example `www-data` or `apache`, should own the folder where you place the viewer. Settings for configuring the viewer are found in the `viewer.html` file. The default settings work with the Demo Widget (see below), but should be updated to match your `guides` directory on your server. For example, you will most likely want to store A2J Guided Interview® files in a directory that is populated internally or with your own file upload interface.
+## Installing from zip
+Download the latest viewer from the releases page https://github.com/CCALI/a2jviewer/releases. Unzip the viewer package into your webroot or preferred directory on your web server. The Apache user, for example `www-data` or `apache`, should own the folder where you place the viewer. Settings for configuring the viewer are found in the `viewer.html` file. The default settings work with the Demo Widget (see below), but should be updated to match your `guides` directory on your server. For example, you will most likely want to store A2J Guided Interview® files in a directory that is populated internally or with your own file upload interface.
+
+##Installing from source
+
+### Install build tools
+
+1.)  Install nvm
+The DAT is a simple restful API that requires nodejs to serve endpoints. Though, you are free to install the node version that the DAT targets and manage it manually, the recommended method is to use a node version manager which will allow the simultaneous installation of multiple versions of node and mitigates several administration issues.
+
+Obtain nvm for windows here: https://github.com/coreybutler/nvm-windows
+
+For \*nix go here: https://github.com/creationix/nvm
+
+2.) Install node through nvm
+after installation of nvm, type the following commands in the terminal to install the required node version
+
+```
+nvm install 12.20.1
+nvm use 12.20.1
+```
+
+check that the install was successful by typing
+
+`node -v`
+
+which should produce the version number of node we installed, `12.20.1`
+
+3.) clone this repo (https://github.com/CCALI/a2jviewer/releases)
+
+### Build from source
+
+`npm run deploy`
+
+OR the equivalent
+
+```
+npm install
+npm run build
+mv index.html index.dev.html
+mv index.production.html index.html
+```
 
 ## Upgrading
```

Comment:
maybe make special mention of the special endpoints here?

Reply:
added note in latest commit